### DISCUSSION TITLE
Regarding the payment

### DISCUSSION THREAD

**Discussion by MagnusTato** [Hyperclaw79/PokeBall-SelfBot#257] [Q&A]
Good evening,

I'm interested in donating for the premium version of this poketwo selfbot, but there's one small question i have. Is the payment 25$ **US**? Just making sure, since i live in Canada... Thanks in advance!

**Comment by MagnusTato** [comment]
One more thing: i tried sending this to your email first, but it says the email either does not exist, or doesn't allow any emails to be sent to it...

**Comment by Hyperclaw79** [comment]
Yes, it is $25 USD.

Are you sure you've copied the mail address properly? Pretty sure it is valid as its my primary email lol. No one faced that issue.

**Reply by MagnusTato** [reply]
All i did was click the blue "mail" on the readme and it redirected me to gmail. Is that not the right address?

**Reply by Hyperclaw79** [reply]
Hmm I'm not sure what exactly is happening in your case. I was able to get mails from two buyers today itself the same way.
Let me know what email you see in your Gmail after clicking that button.

**Reply by MagnusTato** [reply]
The email shown is //harshith.thota7@gmail.com

**Reply by Hyperclaw79** [reply]
The `//`, before the actual mail address, just remove them and it should be working.

**Reply by MagnusTato** [reply]
Oooh, i get it now. I'm sorry for the bother! Must seem kinda stupid x)
